pybamm/experiment/experiment.py

```diff
@@ -42,13 +42,17 @@ class Experiment:
 
     def __init__(
         self,
-        operating_conditions,
-        period="1 minute",
-        temperature=None,
-        termination=None,
+        operating_conditions: list,
+        period: str = "1 minute",
+        temperature: float = None,
+        termination: list = None,
```

Comment:
A clever way to account for the `Optional` type:
```suggestion
        temperature: float | None = None,
        termination: list | None = None,
```

Reply:
Also, the `list` type should be more specific - `list[...]`